jre_emul/android/platform/libcore/harmony-tests/src/test/java/org/apache/harmony/tests/java/util/zip/GZIPInputStreamTest.java

```diff
@@ -27,10 +27,17 @@
 import java.util.zip.Checksum;
 import java.util.zip.GZIPInputStream;
 import java.util.zip.GZIPOutputStream;
-
+//import libcore.junit.junit3.TestCaseWithRules;
```

Comment:
Follow the convention commenting out lines. Same below for the rest of the file.

Reply:
added J2ObjC comments